Doc/library/os.rst

```diff
@@ -3781,6 +3781,435 @@ features:
    .. versionadded:: 3.10
 
 
+Timer File Descriptors
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 3.13
+
+These functions provide support for Linux's *timer file descriptor* API.
+Naturally, they are all only available on Linux.
+
+The following example shows how to use a timer file descriptor
+to execute a function twice a second:
+
+.. code-block:: python
+
+   # Practical scripts should use really use a non-blocking timer,
+   # we use a blocking timer here for simplicity.
+   import os, time
+
+   # Create the timer file descriptor
+   fd = os.timerfd_create(time.CLOCK_REALTIME)
```

Comment:
I dislike this clock, it can jump backward/forward at sysadmin will. Why using using MONOTONIC for examples?

Reply:
@vstinner 
1. linux manual uses it as an example.
https://man7.org/linux/man-pages/man2/timerfd_create.2.html

> it can jump backward/forward at sysadmin will

2. Users can detect system clock change by calling timerfd_settime with TFD_TIMER_ABSTIME and TFD_TIMER_CANCEL_ON_SET. If system clock changes, timer will be canceled.
Users need to choose right flag based on the purpose.

Reply:
TFD_TIMER_CANCEL_ON_SET is really nice.

Reply:
@vstinner 
Resolved?